uber/site_sections/jobs.py

```diff
@@ -124,6 +123,14 @@ def form(self, session, message='', **params):
                 tgt_start_time = str(job.start_time_local).replace(" ", "T")
                 raise HTTPRedirect('index?location=' + str(job.location) + '&time=' + tgt_start_time)
 
+        if 'start_time' in params and 'type' not in params:
```

Comment:
This is the most relevant part in the backend. If you decide you'd rather not merge master into your fork, feel free to just copy-paste this and the `dayClick` option from jobs/index and you should be set.